client/src/components/shared/Info/Info.js

```diff
@@ -104,8 +106,8 @@ const Findata = ({data}: {data: FinancialData}) => {
   )
 }
 
-const Info = ({data, canClose, onClose, showOnMap}: InfoProps) => (
-  <Container className={canClose ? 'info closable' : 'info'}>
+const Info = ({data, canClose, onClose, showOnMap, className}: InfoProps) => (
+  <Container className={classnames(className, canClose ? 'info closable' : 'info')}>
```

Comment:
When already using `classnames` utility, why not refactor also `info closable` classes ? Maybe something like
```
classnames('info', className, {closable: canClose})
```